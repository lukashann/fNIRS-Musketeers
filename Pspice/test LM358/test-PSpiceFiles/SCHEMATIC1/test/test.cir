** Profile: "SCHEMATIC1-test"  [ H:\Uni Stuff\Year 3\Project\Pspice\test\test-pspicefiles\schematic1\test.sim ] 

** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of H:\cdssetup\OrCAD_PSpice/16.6.0/PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.DC LIN V_V4 0 5 0.001 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 


.END
