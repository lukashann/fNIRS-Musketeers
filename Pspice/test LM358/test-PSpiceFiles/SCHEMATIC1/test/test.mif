 lib=C:\Cadence\SPB_16.6\tools\PSpice\Library\opamp.lib, offset=54585, size=371
 lib=C:\Cadence\SPB_16.6\tools\PSpice\Library\opamp.lib, offset=53165, size=1419
