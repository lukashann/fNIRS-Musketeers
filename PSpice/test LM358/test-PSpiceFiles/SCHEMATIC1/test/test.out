
**** 11/18/16 14:03:04 ***** PSpice 16.6.0 (October 2012) ***** ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ H:\Uni Stuff\Year 3\Project\Pspice\test\test-pspicefiles\schematic1\test.sim ] 


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of H:\cdssetup\OrCAD_PSpice/16.6.0/PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.DC LIN V_V4 0 5 0.001 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 



**** INCLUDING SCHEMATIC1.net ****
* source TEST
V_V2         N001590 0 5Vdc
V_V3         0 N001771 5Vdc
X_U1         N00705 N00068 N001590 N001771 N00068 LM358 
V_V4         N00705 0 1Vdc

**** RESUMING test.cir ****
.END

**** 11/18/16 14:03:04 ***** PSpice 16.6.0 (October 2012) ***** ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ H:\Uni Stuff\Year 3\Project\Pspice\test\test-pspicefiles\schematic1\test.sim ] 


 ****     Diode MODEL PARAMETERS


******************************************************************************




               X_U1.x_lm358.dx X_U1.x_lm358.dy 
          IS  800.000000E-18  800.000000E-18 
          RS    1               1.000000E-03 
         CJO                   10.000000E-12 


**** 11/18/16 14:03:04 ***** PSpice 16.6.0 (October 2012) ***** ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ H:\Uni Stuff\Year 3\Project\Pspice\test\test-pspicefiles\schematic1\test.sim ] 


 ****     BJT MODEL PARAMETERS


******************************************************************************




               X_U1.x_lm358.qx 
               PNP             
       LEVEL    1            
          IS  800.000000E-18 
          BF  166.7          
          NF    1            
          BR    1            
          NR    1            
         ISS    0            
          RE    0            
          RC    0            
         CJE    0            
         VJE     .75         
         CJC    0            
         VJC     .75         
         MJC     .33         
        XCJC    1            
         CJS    0            
         VJS     .75         
          KF    0            
          AF    1            
          CN    2.2          
           D     .52         


**** 11/18/16 14:03:04 ***** PSpice 16.6.0 (October 2012) ***** ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ H:\Uni Stuff\Year 3\Project\Pspice\test\test-pspicefiles\schematic1\test.sim ] 


 ****     Current Controlled Switch MODEL PARAMETERS


******************************************************************************




               X_U1.x_lm358.sw2 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 


               X_U1.x_lm358.sw1 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U1.x_lm358.sw3 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U1.x_lm358.sw4 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 



          JOB CONCLUDED

**** 11/18/16 14:03:04 ***** PSpice 16.6.0 (October 2012) ***** ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ H:\Uni Stuff\Year 3\Project\Pspice\test\test-pspicefiles\schematic1\test.sim ] 


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  Total job time (using Solver 1)   =         .22
